### PR TITLE
Relay electron exit code

### PR DESCRIPTION
**Pull request by fasterthanlime**
**TL;DR: The current `cli.js` wrapper always exits with `0`, no matter what electron actually exits with. This PR fixes that.**

Real-life example: some applications (like electron itself) have an interesting test setup:

  * electron is launched, from which
  * ...a BrowserWindow is created, in which
  * ...some javascript is loaded (e.g. mocha + a bunch of specs)
  * The test results (number of tests failed, for instance) are [sent by ipc](https://github.com/atom/electron/blob/399f47ef0f0017a1ad52fab2399b1e1ef5103e29/spec/static/index.html#L73) back to the nodejs side
  * ...where [`process.exit()` is called](https://github.com/atom/electron/blob/399f47ef0f0017a1ad52fab2399b1e1ef5103e29/spec/static/main.js#L24) so that test failures result in a non-zero exit code
  * which is useful when running on [Travis CI](https://travis-ci.org/) by example.

Now, [atom/electron](https://travis-ci.org/atom/electron) isn't touched by this, since they also build a fresh copy of electron right there on the Travis CI instance so I'm assuming they run the binary directly.

However, if anyone wants to use `electron-prebuilt` to run tests on Travis CI via the same method, they won't be able to detect failures, as [this build](https://travis-ci.org/itchio/itchio-app/builds/83977004#L519) demonstrates (It should definitely fail).

This PR fixes that in what I'm hoping is the least possible amount of changes, and I can't think of a way it would break electron-prebuilt for anyone else. Let me know, though!

### CODE REVIEW

**Comment by mafintosh** [comment]
Looks good :+1:

**Comment by mafintosh** [comment]
Will be automatically released next time there is an electron release

**Comment by fasterthanlime** [comment]
Fantastic, thanks for the record time merge!
